dashboard/nasa_pipeline.py

```diff
@@ -23,9 +23,19 @@ def get_image_of_the_day() -> str:
         with conn.cursor() as cur:
             cur.execute(query)
             data = cur.fetchone()
-            print(data)
     return data
 
+def get_moon_phase() -> str:
+    '''Returns todays moon phase'''
+    query = """SELECT image_url FROM image
+    WHERE image_name = 'moon_phase' AND
+    image_date = CURRENT_DATE"""
```

Comment:
Nice simple query, no need to overcomplicate it, thanks to our streamlined database no doubt